crates/wallet/src/descriptor/policy.rs

```diff
@@ -952,11 +999,14 @@ impl<Ctx: ScriptContext + 'static> ExtractPolicy for Miniscript<DescriptorPublic
                 Some(policy)
             }
             Terminal::Older(value) => {
-                let mut policy: Policy = SatisfiableItem::RelativeTimelock { value: *value }.into();
+                let mut policy: Policy = SatisfiableItem::RelativeTimelock {
+                    value: (*value).into(),
+                }
+                .into();
```

Comment:
What do you think about having `SatisfiableItem::RelativeTimelock` hold a value of type `relative::LockTime` instead of `Sequence` ?

Reply:
Keep in mind that this module will likely be deprecated in the next major release (2.0) in favor of the new miniscript [plan module](https://docs.rs/miniscript/latest/miniscript/plan/index.html). Unless we're fixing a bug it may not be worth the refactoring to improve the API.

Reply:
@ValuedMammal I think that's a great suggestion, but I'm now unsure if it's worth it considering what @notmandatory, I'll take a look if it's only a few LOC/changes that would be needed for the improvement.

Reply:
@ValuedMammal I updated the variant, it was trivial. I think the most of work would be updating `Condition` (below), but I don't its work as @notmandatory  mentioned 🤔  https://github.com/bitcoindevkit/bdk/blob/473ef9714f642ee2ad393541763881cf69132262/crates/wallet/src/descriptor/policy.rs#L446-L456

Reply:
`Condition` should be fine as it is now